pytest_mpi.py

```diff
@@ -174,6 +178,18 @@ def _set_parallel_callback(item):
     if nprocs == 1:
         return
 
+    # Set the executable by sniffing sys.argv[0]
```

Comment:
```suggestion
    # Set the executable by sniffing sys.argv[0]
    # This is necessary since invoking pytest in different ways leads to different behaviour:
    # https://docs.pytest.org/en/latest/how-to/usage.html#calling-pytest-through-python-m-pytest
```